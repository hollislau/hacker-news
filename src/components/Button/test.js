import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Button from '.';

describe('Button', () => {
  it('renders', () => {
    const div = document.createElement('div');
    const onClick = () => {};

    ReactDOM.render(
      <Button onClick={ onClick }>
        Button
      </Button>,
      div
    );
  });

  test('snapshots', () => {
    const onClick = () => {};
    const component = renderer.create(
      <Button onClick={ onClick }>
        Button
      </Button>
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
