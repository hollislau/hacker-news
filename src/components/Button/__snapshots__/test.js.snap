exports[`Button snapshots 1`] = `
<button
  className=""
  onClick={[Function]}
  type="button">
  Button
</button>
`;
